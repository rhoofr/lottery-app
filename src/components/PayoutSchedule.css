.card__title-payout {
  margin-bottom: 2rem;
  margin-top: 0;
}

.table-header__2 {
  text-align: left;
}

.table-header__2 th {
  padding-left: 2rem;
}

.payout {
  padding: 0.5rem 0.5rem 0.5rem 2rem !important;
  text-align: left;
}

.jackpot--table {
  margin-bottom: 1.5rem;
}
/* .p-dialog-header-payout {
  padding-bottom: 0 !important;
} */

.payout-notes {
  display: inline-block;
  width: 50%;
}

.payout-notes__details {
  font-size: 1rem;
}

.reg-number {
  display: inline-block;
  border: 2px solid black;
  border-radius: 50%;
  height: 1.3rem;
  width: 1.3rem;
  margin-right: 0.7rem;
}

.ball-number {
  display: inline-block;
  border-radius: 50%;
  height: 1.3rem;
  width: 1.3rem;
  background-color: red;
}
