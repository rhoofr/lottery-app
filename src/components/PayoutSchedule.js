import React from 'react';

import './PayoutSchedule.css';

function PayoutSchedule() {
  return (
    <div>
      <h3 className='card__title-payout'>PAYOUT SCHEDULE</h3>
      <table className='table table-striped table-bordered border-primary jackpot--table'>
        <thead className='table-light'>
          <tr key={1}>
            <th scope='col' colSpan={2}>
              Powerball *
            </th>
            <th scope='col' colSpan={2}>
              Mega Millions **
            </th>
          </tr>
          <tr key={2} className='table-header__2'>
            <th scope='col'>Match</th>
            <th scope='col'>Payout</th>
            <th scope='col'>Match</th>
            <th scope='col'>Payout</th>
          </tr>
        </thead>
        <tbody>
          <tr key={3}>
            <td className='payout'>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='ball-number'></div>
            </td>
            <td className='payout'>Jackpot</td>
            <td className='payout'>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='ball-number'></div>
            </td>
            <td className='payout'>Jackpot</td>
          </tr>
          <tr key={4}>
            <td className='payout'>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
            </td>
            <td className='payout'>$1,000,000</td>
            <td className='payout'>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
            </td>
            <td className='payout'>$1,000,000</td>
          </tr>
          <tr key={5}>
            <td className='payout'>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='ball-number'></div>
            </td>
            <td className='payout'>$50,000</td>
            <td className='payout'>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='ball-number'></div>
            </td>
            <td className='payout'>$10,000</td>
          </tr>
          <tr key={6}>
            <td className='payout'>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
            </td>
            <td className='payout'>$100</td>
            <td className='payout'>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
            </td>
            <td className='payout'>$500</td>
          </tr>
          <tr key={7}>
            <td className='payout'>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='ball-number'></div>
            </td>
            <td className='payout'>$100</td>
            <td className='payout'>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='ball-number'></div>
            </td>
            <td className='payout'>$200</td>
          </tr>
          <tr key={8}>
            <td className='payout'>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
            </td>
            <td className='payout'>$7</td>
            <td className='payout'>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
            </td>
            <td className='payout'>$10</td>
          </tr>
          <tr key={9}>
            <td className='payout'>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='ball-number'></div>
            </td>
            <td className='payout'>$7</td>
            <td className='payout'>
              <div className='reg-number'></div>
              <div className='reg-number'></div>
              <div className='ball-number'></div>
            </td>
            <td className='payout'>$10</td>
          </tr>
          <tr key={10}>
            <td className='payout'>
              <div className='reg-number'></div>
              <div className='ball-number'></div>
            </td>
            <td className='payout'>$4</td>
            <td className='payout'>
              <div className='reg-number'></div>
              <div className='ball-number'></div>
            </td>
            <td className='payout'>$4</td>
          </tr>
          <tr key={11}>
            <td className='payout'>
              <div className='ball-number'></div>
            </td>
            <td className='payout'>$4</td>
            <td className='payout'>
              <div className='ball-number'></div>
            </td>
            <td className='payout'>$2</td>
          </tr>
        </tbody>
      </table>
      <div className='payout-notes'>
        <span className='payout-notes__details'>
          * Powerball (Regular Numbers 1-69, Ball Numbers 1-26)
        </span>
      </div>
      <div className='payout-notes'>
        <span className='payout-notes__details'>
          ** Mega Millions (Regular Numbers 1-70, Ball Numbers 1-25)
        </span>
      </div>
    </div>
  );
}

export default PayoutSchedule;
