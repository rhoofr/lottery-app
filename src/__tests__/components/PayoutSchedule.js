import React from 'react';
import { shallow } from 'enzyme';
import PayoutSchedule from '../../components/PayoutSchedule';

describe('--- PayoutSchedule snapshot ---', () => {
  test('should render PayoutSchedule Page correctly', () => {
    const wrapper = shallow(<PayoutSchedule />);
    expect(wrapper).toMatchSnapshot();
  });
});
