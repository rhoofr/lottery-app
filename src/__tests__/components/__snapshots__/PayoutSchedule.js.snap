// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`--- PayoutSchedule snapshot --- should render PayoutSchedule Page correctly 1`] = `
<div>
  <h3
    className="card__title-payout"
  >
    PAYOUT SCHEDULE
  </h3>
  <table
    className="table table-striped table-bordered border-primary jackpot--table"
  >
    <thead
      className="table-light"
    >
      <tr
        key="1"
      >
        <th
          colSpan={2}
          scope="col"
        >
          Powerball *
        </th>
        <th
          colSpan={2}
          scope="col"
        >
          Mega Millions **
        </th>
      </tr>
      <tr
        className="table-header__2"
        key="2"
      >
        <th
          scope="col"
        >
          Match
        </th>
        <th
          scope="col"
        >
          Payout
        </th>
        <th
          scope="col"
        >
          Match
        </th>
        <th
          scope="col"
        >
          Payout
        </th>
      </tr>
    </thead>
    <tbody>
      <tr
        key="3"
      >
        <td
          className="payout"
        >
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="ball-number"
          />
        </td>
        <td
          className="payout"
        >
          Jackpot
        </td>
        <td
          className="payout"
        >
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="ball-number"
          />
        </td>
        <td
          className="payout"
        >
          Jackpot
        </td>
      </tr>
      <tr
        key="4"
      >
        <td
          className="payout"
        >
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
        </td>
        <td
          className="payout"
        >
          $1,000,000
        </td>
        <td
          className="payout"
        >
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
        </td>
        <td
          className="payout"
        >
          $1,000,000
        </td>
      </tr>
      <tr
        key="5"
      >
        <td
          className="payout"
        >
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="ball-number"
          />
        </td>
        <td
          className="payout"
        >
          $50,000
        </td>
        <td
          className="payout"
        >
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="ball-number"
          />
        </td>
        <td
          className="payout"
        >
          $10,000
        </td>
      </tr>
      <tr
        key="6"
      >
        <td
          className="payout"
        >
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
        </td>
        <td
          className="payout"
        >
          $100
        </td>
        <td
          className="payout"
        >
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
        </td>
        <td
          className="payout"
        >
          $500
        </td>
      </tr>
      <tr
        key="7"
      >
        <td
          className="payout"
        >
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="ball-number"
          />
        </td>
        <td
          className="payout"
        >
          $100
        </td>
        <td
          className="payout"
        >
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="ball-number"
          />
        </td>
        <td
          className="payout"
        >
          $200
        </td>
      </tr>
      <tr
        key="8"
      >
        <td
          className="payout"
        >
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
        </td>
        <td
          className="payout"
        >
          $7
        </td>
        <td
          className="payout"
        >
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
        </td>
        <td
          className="payout"
        >
          $10
        </td>
      </tr>
      <tr
        key="9"
      >
        <td
          className="payout"
        >
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="ball-number"
          />
        </td>
        <td
          className="payout"
        >
          $7
        </td>
        <td
          className="payout"
        >
          <div
            className="reg-number"
          />
          <div
            className="reg-number"
          />
          <div
            className="ball-number"
          />
        </td>
        <td
          className="payout"
        >
          $10
        </td>
      </tr>
      <tr
        key="10"
      >
        <td
          className="payout"
        >
          <div
            className="reg-number"
          />
          <div
            className="ball-number"
          />
        </td>
        <td
          className="payout"
        >
          $4
        </td>
        <td
          className="payout"
        >
          <div
            className="reg-number"
          />
          <div
            className="ball-number"
          />
        </td>
        <td
          className="payout"
        >
          $4
        </td>
      </tr>
      <tr
        key="11"
      >
        <td
          className="payout"
        >
          <div
            className="ball-number"
          />
        </td>
        <td
          className="payout"
        >
          $4
        </td>
        <td
          className="payout"
        >
          <div
            className="ball-number"
          />
        </td>
        <td
          className="payout"
        >
          $2
        </td>
      </tr>
    </tbody>
  </table>
  <div
    className="payout-notes"
  >
    <span
      className="payout-notes__details"
    >
      * Powerball (Regular Numbers 1-69, Ball Numbers 1-26)
    </span>
  </div>
  <div
    className="payout-notes"
  >
    <span
      className="payout-notes__details"
    >
      ** Mega Millions (Regular Numbers 1-70, Ball Numbers 1-25)
    </span>
  </div>
</div>
`;
